### PR TITLE
Allow ability to disable menu items

### PR DESCRIPTION
**Pull request by jl-gogovapps**
Exposes an interface to disable items 🤷 
```
  disabledToolbarItems?: Array<string> | undefined;
  disabledBlockMenuItems?: Array<string> | undefined;
```

Certainly open to feedback here and thoughts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "rich-markdown-editor",
+  "name": "@gogovapps/rich-markdown-editor",
```
Comment (jl-gogovapps) [inline package.json:2]: This was so I could publish it to my repo

**Comment by viperfx** [comment]
Would love to have this merged in. @tommoor This would be useful.

**Comment by saimonmoore** [comment]
This is an interesting feature but I'm noticing very little communication on the few PR's that are open? e.g. This PR has been open for 6 months.

**Comment by tommoor** [comment]
Hi. If a feature like this were to be merged it would need to be more complete in it's removal of the individual content types - for example although this allows disabling icons in the toolbar it does not disable the associated keyboard shortcut or the markdown syntax for inserting those items.

If you need a flexible editor of this nature I would recommend https://github.com/remirror/remirror

**Comment by saimonmoore** [comment]
Ok that's very good feedback indeed @tommoor .  Thanks for that and thanks for the pointer to remirror. I'm looking for a simple, flexible editor that reads and persists markdown. I'll take a good long look at it to see if it works (Still in beta but looks very promising).

**Comment by viperfx** [comment]
That's really too bad, as I think most of us are looking for a simple markdown editor that is quite clean without any toolbars but has live preview similar to the outline editor.

Having a config for the menu items would be awesome - and again my use case would be to take out features to match exactly what markdown standard supports, and take out anything outline specific - rather than add anything new in. 

I hope you do consider this again @tommoor

**Comment by tommoor** [comment]
> I hope you do consider this again @tommoor

Not against it in theory, just needs to be properly done to remove the entire node type from the editor rather than just the toolbar icon.

**Comment by viperfx** [comment]
and just to understand you have no plans to support this internally @tommoor  - this would only possible through a community PR?

**Comment by tommoor** [comment]
Thanks for the effort. This functionality was included as part of above PR ^

**Pull request closed without merging**